lib/rex/post/meterpreter/command_mapper.rb

```diff
@@ -97,6 +98,30 @@ def self.get_commands(*extensions)
     commands
   end
 
+  def self.create_tlv_types_cache
```

Comment:
Since, for now, this can return an array

```suggestion
  def self.get_tlv_names(id)
```

Reply:
It'd be great to sprinkle some yard doc metadata here, specifically the `@param` and `@return` metadata with a link to https://github.com/rapid7/metasploit-framework/issues/16267 explaining why it can return an array

Context: https://github.com/lsegal/yard